DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml

```diff
@@ -30,7 +30,8 @@
   {
     <tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
       <tr role="row" class="nhsuk-table__row first-row">
-        <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
+        <td role="cell" rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
+          <partial name="_ComperencyFlags" model="competency.CompetencyFlags" />
```

Comment:
When rendered here, the flag container could do with `nhsuk-u-padding-left-4` class adding so that they align with the competency beneath them, which is indented from the left:
![image](https://user-images.githubusercontent.com/67740339/174088212-2ad0a3ca-3b1e-4d8b-9821-d1c345463136.png)

Reply:
Added new commit.